contrib/golang/filters/http/source/config.cc

```diff
@@ -33,7 +35,11 @@ Http::FilterFactoryCb GolangFilterConfig::createFilterFactoryFromProtoTyped(
       proto_config, dso_lib, fmt::format("{}golang.", stats_prefix), context);
   config->newGoPluginConfig();
   return [config, dso_lib](Http::FilterChainFactoryCallbacks& callbacks) {
-    auto filter = std::make_shared<Filter>(config, dso_lib);
+    const std::string& worker_name = callbacks.dispatcher().name();
+    auto pos = worker_name.find_first_of('_');
+    ENVOY_BUG(pos != std::string::npos, "worker name is not in expected format worker_{id}");
+    const uint32_t worker_id = std::stoi(worker_name.substr(pos + 1));
```

Comment:
sorry should've caught this earlier. Don't use std::stoi on the data plane: it can throw an exception which will not be caught. 

Can you instead call absl::SimpleAtoi -- you can use ENVOY_BUG on the bool return.

Reply:
fixed!